### ISSUE TITLE
Expo Android crashes on this invalid stylesheet

### ISSUE THREAD

**Issue by libjared** [expo/expo#766]
Within my first 5 minutes of trying React Native, I've encountered a crash. I was able to reproduce it in an Expo Snack. I am running Oreo on a Google Pixel. I am totally new to React Native and Expo, so if this is an upstream issue, needs more information, or is expected behavior, please close this. But I would typically expect any app to gracefully show an error rather than crash.

https://snack.expo.io/HybVjhs3b

In a nutshell, create a View that uses a StyleSheet. In the stylesheet, when 'padding' or its top/bottom/etc counterparts are set to a string instead of a number, the Expo app crashes.

**Comment by ccheever** [comment]
Hi-

Yeah. You need to give a number for those values. I think this is an issue with React Native (upstream) rather than Expo; it would probably make sense to file an issue here: https://github.com/facebook/react-native/issues
